plugins/branch/branch.plugin.zsh

```diff
@@ -0,0 +1,26 @@
+# Branch: displays the current Git or Mercurial branch fast.
+# Victor Torres <vpaivatorres@gmail.com>
+# Oct 2, 2015
+
+function branch_prompt_info() {
+  # Defines path as current directory
+  path=$(pwd)
+  # While current path is not root path
+  while [ $path != '/' ];
+  do
+    # Git repository
+    if [ -d ${path}/.git ];
```

Comment:
Wrap the whole thing inside double quotes in case there are spaces, otherwise there's a syntax error: `[[ -d "$var/.git" ]]`